.github/workflows/ci_tests.yml

```
@@ -237,8 +240,12 @@ jobs:
             ENABLE_HYPRE_DEVICE: CUDA
             ENABLE_HYPRE: ON
             ENABLE_TRILINOS: OFF
-            RUNS_ON: Runner_8core_32GB
-
+            RUNS_ON: streak2
+            NPROC: 16
+            DOCKER_RUN_ARGS: "--cpus=16 --memory=192g --runtime=nvidia --gpus all"
+            DOCKER_CERTS_DIR: "/usr/local/share/ca-certificates"
+            DOCKER_CERTS_UPDATE_COMMAND: "update-ca-certificates"
```

Is it possible to get the bundle of certificates from `streak*` into the container by using a _read-only_ `mount` `docker` approach?

If we do something like `docker run -v /path/to/llnl.crt.pem:/usr/local/share/ca-certificates/llnl/bundle.crt:ro` (note the trailing `ro`), then simply running `update-ca-certificates` or `update-ca-trust` (se can let the distribution decide what's available) would do the job?

So we could reduce those lines to:
```suggestion
            DOCKER_RUN_ARGS: "--cpus=16 --memory=192g --runtime=nvidia --gpus all -v /path/to/llnl.crt.pem:/usr/local/share/ca-certificates/llnl/bundle.crt:ro"
```
and remove the copy part (and everything related to the `hostname`) from `scripts/ci_build_and_test_in_container.sh`?

---

yeah. I forgot to do that...

---

There are several certificate files to "copy", and they need their extension changed. So I think the the "copy" is preferable to the volume mounting of specific files.

---

From the doc I could see that
```
update-ca-certificates  is a program that updates the directory [/etc/ssl/certs](file:///etc/ssl/certs) to hold SSL
       certificates  and  generates  ca-certificates.crt,  a  concatenated  single-file  list  of
       certificates.
```
So could we take that `ca-certificates.crt` and mount it like `-v /path/to/ca-certificates.crt:/somewhere/llnl.pem:ro` it should be enough? We can mount one file with another name/extension. That would spare us the copy burden?